src/coreclr/build-test.cmd

```diff
@@ -506,21 +506,6 @@ echo { "build_os": "%__BuildOS%", "build_arch": "%__BuildArch%", "build_type": "
 
 :SkipBuildingWrappers
 
-REM =========================================================================================
-REM ===
-REM === Copy CoreFX assemblies if needed.
-REM ===
-REM =========================================================================================
-
-if NOT "%__LocalCoreFXPath%"=="" (
-    echo Patch CoreFX from %__LocalCoreFXPath% ^(%__LocalCoreFXConfig%^)
-    set NEXTCMD=python "%__ProjectDir%\tests\scripts\patch-corefx.py" -clr_core_root "%CORE_ROOT%"^
```

Comment:
Is `patch-corefx.py` script still used for anything? If not, delete it?

Reply:
Jeremy deleted the script as part of his change. The only trouble was that, as my change was moving this block to another place in the script to fix step ordering, git didn't realize it's a conflict that would need resolving and instead treated my change as adding this section in a different place. It can never be hit today as the logic for setting __LocalCoreFXPath no longer exists (that was also deleted by Jeremy).